generate-service-descriptor.hs

```diff
@@ -0,0 +1,47 @@
+import System.Environment
+
+serviceDescriptor :: String -> String
+serviceDescriptor version = "{\n\
+\   \"description\": \"Eval stuff\",\n\
+\   \"name\": \"hiplambda\",\n\
+\   \"organization\": \"RD:Identity\",\n\
+\   \"externalAccess\": false,\n\
+\   \"resources\": [{\n\
+\      \"type\": \"postgres-db\",\n\
+\      \"name\": \"db\"\n\
+\   }],\n\
+\   \"links\": {\n\
+\      \"binary\": {\n\
+\         \"name\": \"docker.atlassian.io/atlassian/hiplambda\",\n\
+\         \"type\": \"docker\",\n\
+\         \"tag\": \"" ++ version ++ "\"\n\
+\      },\n\
+\      \"healthcheck\": {\n\
+\         \"uri\": \"heartbeat\"\n\
+\      },\n\
+\      \"source\": {\n\
+\         \"url\": \"https://github.com/purefn/hiplambda.git\"\n\
+\      }\n\
+\   },\n\
+\   \"owners\": [\n\
+\      \"aknoll@atlassian.com\"\n\
```

Comment:
The owner is the person who is going to be contacted if there are alerts or if the micros team needs to know who to contact about a service.